deltametrics/section.py

```diff
@@ -912,57 +1268,146 @@ class CircularSection(BaseSection):
         section, or the `Cube` is passed as the first positional argument
         during instantiation.
 
+    .. warning::
```

Comment:
this looks right, but didn't render as expected 
![image](https://user-images.githubusercontent.com/1770513/143049358-9931a36e-534f-4e8c-9dfe-9e6aa3787c55.png)